python/test_galario.py

```diff
@@ -488,3 +488,10 @@ def test_loss(nsamples, real_type, complex_type, rtol, atol, acc_lib, pars):
     # assert_allclose(fint_shifted.real, sampled.real, rtol, atol)
     # assert_allclose(fint_shifted.imag, sampled.imag, rtol, atol)
 
+def test_exception():
+    """
+    Make sure exceptions propagate from C++ to python
+    """
+    with pytest.raises(ValueError, message="Odd image lengths are not permitted"):
```

Comment:
How is the C++ error type translated into Python error? Is there a reference for this?

Reply:
Yes, http://cython.readthedocs.io/en/latest/src/userguide/wrapping_CPlusPlus.html?highlight=RuntimeError#exceptions